openwisp_radius/migrations/0004_default_permissions.py

```diff
@@ -0,0 +1,54 @@
+from django.db import migrations
+from django.contrib.auth.models import Permission
+
+from . import create_default_permissions
+
+
+def assign_permissions_to_groups(apps, schema_editor):
+    create_default_permissions(apps, schema_editor)
+    Group = apps.get_model('openwisp_users', 'Group')
+    admin = Group.objects.get(name='Administrator')
+    operator = Group.objects.get(name='Operator')

```

Comment:
see this commit and apply the same change please: https://github.com/openwisp/openwisp-controller/commit/d08d07598ab526d81a1d5bc55dca1f9f34476111